spec/namespaces/sql.yaml

```
@@ -24,7 +24,7 @@ paths:
       operationId: sql.query.0
       x-operation-group: sql.query
       x-version-added: '1.0'
-      description: Send a SQL/PPL query to the SQL plugin. 
+      description: Executes SQL or PPL queries against OpenSearch indices.
```

```suggestion
      description: Executes SQL or PPL queries against OpenSearch indexes.
```

I know .. but that's the convention.

---

Hm, these changes were not intended to be part of my PR . Forgot to rebase main. Updated spellings nonetheless!